Documentation/articles/getting_started/3_adding_content.md

```diff
@@ -22,11 +22,11 @@ Now open up your game project and look at the Solution explorer window. Expand t
 
 ![Open Content](~/images/getting_started/3_open_content.png)
 
-You should now see the Pipeline Editor window open up. If it does not open up (you see a text file open), then you can right-click on **Content.mgcb** and select **Open With**, then select **MonoGame Pipeline** in the list, click **Set as Default** and then click **OK**.
+You should now see the MGCB Editor window open up. If it does not open up (you see a text file open), then you can right-click on **Content.mgcb** and select **Open With**, then select **mgcb-editor-wpf** in the list, click **Set as Default** and then click **OK**.
```

Comment:
Hmm, this is not a very friendly name. I think we should be able to change this, but I don't know how VS picks this.

Reply:
You mean `mgcb-editor-wpf` ?

@harry-cpp ?

Reply:
Yes

Reply:
Well if you can change it, then that PR should also update the name in the docs, shouldn't affect this PR.

Should be raised as a separate issue if need be (I agree, it's a crap name, seems to be based on the .NET project name maybe?)

Reply:
It's the assembly name and the executable name.

Reply:
My initial thought was that each platform's assembly could be in a subfolder, allowing them all to be named `mgcb-editor`, but then all supporting assemblies, templates, etc. would have to be duplicated in each platform's subfolder. I couldn't think of a clean way to do this so I just gave them unique names so they could live in the same main folder.

Reply:
Yes, we shouldn't change that, just need a way to set the name as it appears in VS. If it's the executable name we can create a shortcut and register that in the Windows registry. Can we reuse the link from the start menu (#7223)?